boards/posix/native_posix/fuse_flash.c

```diff
@@ -0,0 +1,531 @@
+/*
+ * Copyright (c) 2019 Jan Van Winkel <jan.van_winkel@dxplore.eu>
```

Comment:
After a quick look it would seem this file is not really part of the native_posix board, but could be shared by any posix arch based board, and could therefore have been anywhere else(?). If that is the case maybe it would be nicer to place it with the flash driver itself(?)
(The Kconfig option and cmake paragraph would go with it)

Reply:
Good point but would it not be better to move it then to subsys/fs?

Reply:
I have no opinion, I think it would be best if one of the flash or fs maintainers would comment here.

Reply:
^^^ @nashif @ramakrishnapallala